examples/xt.lua

```diff
@@ -0,0 +1,49 @@
+--
+-- SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+-- SPDX-License-Identifier: MIT OR GPL-2.0-only
+--
+
+-- Match DNS packets using netfilter match
```

Comment:
I would rename the file accordingly.. instead of calling this `xt.lua`.. let's avoid such generic names.. also, I would expect this example to be more "useful"..  instead of simply matching proto and ports, do something that you can't do simply using iptables rules.. for instance, parse something on the DNS request and block it accordingly to a blocklist based on such parameter..